onyxia-api/src/main/java/fr/insee/onyxia/api/controller/pub/HealthcheckController.java

```diff
@@ -0,0 +1,42 @@
+package fr.insee.onyxia.api.controller.pub;
+
+import fr.insee.onyxia.api.configuration.CatalogWrapper;
+import fr.insee.onyxia.api.configuration.Catalogs;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import java.util.List;
+import java.util.stream.Collectors;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@Tag(name = "Public")
+@RequestMapping("/public")
+public class HealthcheckController {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(HealthcheckController.class);
+
+    @Autowired private Catalogs catalogs;
+
+    @GetMapping("/healthcheck")
+    public ResponseEntity healthcheck() {
```

Comment:
Great seeing this added in the health check. If it is not to much to ask for I think a unit test covering some common scenarios would be very useful to verify that this code work as intended. Both now and in the future if/when changes will occur.